app/root.css

```diff
@@ -164,22 +152,24 @@ h3 a:visited {
   cursor: pointer;
 }
 
-.icon-link-group svg,
-.actions img {
+.icon-link-group svg {
   width: 32px;
   height: 32px;
-  filter: var(--light, invert(0)) var(--dark, invert(1));
 }
+
+.actions svg {
+  width: 24px;
+  height: 24px;
+}
+
+.actions svg:not(:hover) *:not(path[style*='fill:#333']),
 .icon-link-group .icon-link:not(:hover) svg *:not(path[style*='fill:#333']) {
   fill: none !important;
 }
-.icon-link-group .icon-link:not(:hover) svg path[style*='stroke'] {
-  stroke: #333 !important;
-  stroke-width: 10 !important;
-}
 
-.icon-link {
-  transition: all 1s ease;
+.actions svg:not(:hover),
+.icon-link-group .icon-link:not(:hover) svg {
+  filter: var(--light, invert(0)) var(--dark, invert(1));
 }
```

Comment:
Does this look ok to you? Any ideas why I have to reload the page a few times before the outlined action icons show up. I shouldn't merge this into the main branch until that gets sorted out.

Reply:
Hm, not sure, I will investigate the action icons hover.

Reply:
Looks like I added the `link-icon` class to svg element itself by accident, should only be on the parent `a` and `button` elements, and then in CSS detect the hover on that parent: `.link-icon:hover svg`. Now updated to show only the outline without hover (and outline is inverted in dark mode). I also removed the inverted colors on hover in dark mode:

<img width="104" alt="Screenshot 2022-07-05 at 10 39 38" src="https://user-images.githubusercontent.com/1087670/177287915-8cd4f716-d4d8-43aa-8956-113981558701.png">

Reply:
actually, I was still able to reproduce the problem that I don't see the outline after I reload the page without any question open, then open a question :/

hmm, I need to investigate more

Reply:
fixed in https://github.com/StampyAI/stampy-ui/pull/48/commits/efbb90b32357d9324ce4c56a91baefe6258dc332